inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/slice/slice_kernel_ref.cpp

```diff
@@ -0,0 +1,107 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include"slice_kernel_ref.h"
+#include <kernel_selector_utils.h>
+#include <vector>
+
+namespace {
+
+void addJitConstantsForAttribute(kernel_selector::JitConstants &jit,
+        const std::string &name, const std::vector<std::int32_t> &attribute) {
+    using namespace kernel_selector;
+    jit.AddConstant(MakeJitConstant(name + "_BATCH", attribute[0]));
+    jit.AddConstant(MakeJitConstant(name + "_FEATURE", attribute[1]));
+    if (attribute.size() == 5) {  // BFZYX
+        jit.AddConstant(MakeJitConstant(name + "_Z", attribute[2]));
+        jit.AddConstant(MakeJitConstant(name + "_Y", attribute[3]));
+        jit.AddConstant(MakeJitConstant(name + "_X", attribute[4]));
+    } else {  // BFYX
+        jit.AddConstant(MakeJitConstant(name + "_Y", attribute[2]));
+        jit.AddConstant(MakeJitConstant(name + "_X", attribute[3]));
+    }
+}
+
+} // anonymous namespace
+
+namespace kernel_selector {
+
+KernelsData SliceKernelRef::GetKernelsData(const Params &params,
+        const optional_params &options) const {
+    if (!Validate(params, options)) {
+        return {};
+    }
+    KernelData kernel_data = KernelData::Default<slice_params>(params);
+    slice_params &new_params =
+            dynamic_cast<slice_params&>(*kernel_data.params.get());
+    auto dispatch_data = SetDefault(new_params, options);
+    auto entry_point = GetEntryPoint(kernelName, new_params.layerID, params, options);
+    auto slice_specific_jit = GetJitConstants(new_params);
+    auto jit = CreateJit(kernelName, slice_specific_jit, entry_point);
+
+    FillCLKernelData(kernel_data.kernels[0], dispatch_data, params.engineInfo,
+            kernelName, jit, entry_point);
+
+    return {kernel_data};
+}
+
+KernelsPriority SliceKernelRef::GetKernelsPriority(const Params&/*params*/,
+        const optional_params&/*options*/) const {
+    return FORCE_PRIORITY_1;
+}
+
+ParamsKey SliceKernelRef::GetSupportedKey() const {
+    ParamsKey k;
+    k.EnableInputDataType(Datatype::F16);
```

Comment:
what about int8 ? the primitive has i8 and u8 in the impelementation_map

Reply:
Resolved in #8267